microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -27,8 +27,8 @@
 from lisa.util import BadEnvironmentStateException, generate_random_chars
 from microsoft.testsuites.xfstests.xfstests import Xfstests
 
-_scratch_folder = "/root/scratch"
-_test_folder = "/root/test"
+_scratch_folder = "/mnt/scratch"
```

Comment:
I'm not sure the folder exists or not, if the VM size doesn't have a resource disk.

Reply:
`/mnt` should be present in all standard linux distros, as it's part of the standard linux file hierarchy.  If VM doesn't have a resource disk there would be no mounts I think, but the directory will exist. For BSD this might be different, which BSD distros we are supporting? I can check for the specific BSD distro's default file hierarchy.

Reply:
Thank you, please have a check on FreeBSD.